lua/user/plugins.lua

```diff
@@ -169,15 +156,16 @@ M.config = function()
         require("user.rust_tools").config()
       end,
       ft = { "rust", "rs" },
-      disable = not lvim.builtin.rust_programming.active,
+      enabled = lvim.builtin.rust_programming.active,
     },
     {
-      "https://git.sr.ht/~whynothugo/lsp_lines.nvim",
-      config = function()
+      "git@git.sr.ht:~whynothugo/lsp_lines.nvim",
```

Comment:
you have to use url =

Reply:
Oh good catch.

Reply:
That broke everything for me 😅

Reply:
mhh looks like it didn't work.
in a other config       url = "https://git.sr.ht/~whynothugo/lsp_lines.nvim",
works

Reply:
Still broken for me.. I'll have to take a deeper look